contributors/design-proposals/raw-block-pv.md

```diff
@@ -0,0 +1,559 @@
+# Local Raw Block Consumption in Kubernetes
+
+Authors: erinboyd@, screeley44@, mtanino@
+
+This document presents a proposal for managing raw block storage in Kubernetes using the persistent volume source API as a consistent model of consumption.
+
+# Terminology
+* Raw Block Device - a physically attached device devoid of a filesystem
+* Raw Block Volume - a logical abstraction of the raw block device as defined by a path
+* File on Block - a formatted (ie xfs) filesystem on top of a raw block device
```

Comment:
I meant that `File on Block` does not make much sense, do you really mean single file on a block devices? Most people use filesystems for that.